tests/lib/rules/visual-label-better-than-aria-suggestion-v9.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/visual-label-better-than-aria-suggestion-v9"),
+    RuleTester = require("eslint").RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+ruleTester.run("visual-label-better-than-aria-suggestion-v9", rule, {
+    valid: [
+        `<><Label id="my-dropdownid">This is the visual label</Label><Dropdown aria-labelledby="my-dropdownid" /></>`,
```

Comment:
I think we can pass in the applicable components to generate the test cases like in this file: input-components-require-accessible-nam